### PR TITLE
goimports update

### PR DESCRIPTION
**Pull request by KashifSaadat**

### CODE REVIEW

File: nodeup/pkg/model/kube_apiserver.go [hunk 1]
```diff
@@ -18,10 +18,11 @@ package model
 
 import (
 	"fmt"
-	"k8s.io/kops/util/pkg/proxy"
 	"path/filepath"
 	"strings"
 
+	"k8s.io/kops/util/pkg/proxy"
```
Comment (gambol99) [inline nodeup/pkg/model/kube_apiserver.go]: So I think the rule of thumb is 
```
(
   stdlib

   local packages

   externals
)
```

Comment (KashifSaadat) [reply nodeup/pkg/model/kube_apiserver.go]: Cleaned up and added it to the tests so it is picked up on all PRs

**Pull request merged**